website/blog/2023-12-21-five-lessons-from-choosing-infrastructure-tooling/item.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "5 lessons from choosing infrastructure tooling for a security startup"
+description: "What's in our stack: the tools we use to build authentik (and why we chose them)."
+slug: 2023-12-21-five-lessons-from-choosing-infrastructure-tooling
+authors:
+    - name: March Schmitt
+      title: Infrastructure Engineer at Authentik Security Inc
+      url: https://github.com/rissson
+      image_url: https://avatars.githubusercontent.com/u/18313093?v=4
+tags:
+    - authentik
+    - startups
+    - infrastructure tooling
+    - tools
+    - technology stack
+    - Loki
+    - Argo CD
+    - Prometheus
+    - Thanos
+    - Transifix
+    - Lit
+    - Redis
+    - Grafana
+    - authentication
+    - Authentik Security
+hide_table_of_contents: false
+image: ./tech-stack.png
+---
+
+> **_authentik is an open source Identity Provider that unifies your identity needs into a single platform, replacing Okta, Active Directory, and auth0. Authentik Security is a [public benefit company](https://github.com/OpenCoreVentures/ocv-public-benefit-company/blob/main/ocv-public-benefit-company-charter.md) building on top of the open source project._**
+
+---
+
+With great power (to choose your own tools) comes great responsibility. Not inheriting a legacy toolchain is an infrastructure engineer’s dream, but it can be hard to know where to start.
+
+As the first infrastructure engineer hired to work on authentik I saw the green field opportunities, but also the responsibility and long-term importance of choosing the best stack of tools and build processes. From my past roles, I already knew many of the considerations that we would need to look closely at.
+
+For example, we know that ease-of-maintenance is a primary consideration, as is the stability and probable longevity of the tool, how well the tools integrate, and of course the level of support we were likely to get for each tool.
+
+In this post we share what we’re using to build authentik, and the lessons behind those choices.
+
+![technology stack for startups](./tech-stack.png)
+
+<!--truncate-->
+
+## #1 Choices are often human, not technical
```

Comment:
```suggestion
# 1. Choices are often human, not technical
```

Reply:
I style the H1s the original way in Docs because otherwise Docs tries to make every subheading a number. I think this looks more natural on a blog, but if Markdown creates the same issues then feel free to revert.

Some of the H1s have been ported over to GitHub as H2s so I'll change these too.

Reply:
Yeah, I don't think we can  use the "#" within the title.. :-( Also, we don't use H1s in the blogs... jump straight to H2s and down from there. It it has been that way forever. Maybe because we want to match the Docs styling.